### PR TITLE
add new Target.getPossibleBreakpointsForMultipleSources command

### PR DESCRIPTION
**Pull request by misterdjules**
Needed for https://github.com/RecordReplay/backend/pull/4179. As mentioned there:

> The current approach is a little bit naive, because it batches all getPossibleBreakpoints messages in a single getPossibleBreakpointsForMultipleSources message. I'm concerned this could lead to blocking the CPU for too long an not letting other messages be processed as quickly as before, having to transfer too much data in one request/response, or other similar problems.

> But since this is my first change that relies on gecko-dev changes, I wanted to get a naive version up first, test that it works and then iterate on it.

Reviewers:
- @kannanvijayan because we talked about trying this out as part of the warm start efforts.
- @loganfsmyth and @bhackett1024 to get their initial feedback on this naive implementation, in case I missed even more potential issues than described above.

### CODE REVIEW

**Review by bhackett1024 — COMMENTED** [review]
This looks good, but if we do the batching at the manifest level (see https://github.com/RecordReplay/backend/pull/4179#issuecomment-1011006349) then changes to gecko and other runtimes won't be necessary, so it would be good to decide whether we want to use that approach first.

**Comment by misterdjules** [comment]
@loganfsmyth @bhackett1024 @kannanvijayan  I think this is ready for another round of reviews. See the [latest updates](https://github.com/RecordReplay/backend/pull/4179#issuecomment-1014881898) to the corresponding RecordReplay/backend PR for more details/

File: devtools/server/actors/replay/module.js [hunk 1]
```diff
@@ -952,6 +953,17 @@ function Debugger_getPossibleBreakpoints({ sourceId, begin, end }) {
   }
 }
 
+function Target_getPossibleBreakpointsForMultipleSources({ sourceIds }) {
+  return { possibleBreakpoints:
+    sourceIds.map((sourceId => {
```
Comment (bhackett1024) [inline devtools/server/actors/replay/module.js]: `possibleBreakpoints:` should go on this line (we don't have eslint running for gecko-dev, alas).

File: devtools/server/actors/replay/module.js [hunk 2]
```diff
@@ -952,6 +953,17 @@ function Debugger_getPossibleBreakpoints({ sourceId, begin, end }) {
...
+      return {
+        sourceId,
+        ...Debugger_getPossibleBreakpoints({sourceId})
```
Comment (bhackett1024) [inline devtools/server/actors/replay/module.js]: This should use a helper function instead of invoking the other handler directly.

**Review by bhackett1024 — APPROVED** [review]

**Comment by misterdjules** [comment]
@bhackett1024 I think I've addressed your comments with the latest commit I just pushed.

**Pull request merged**